internal/app/machined/pkg/runtime/v1alpha1/platform/nocloud/metadata.go

```diff
@@ -58,8 +59,8 @@ type NetworkConfig struct {
 // Ethernet holds network interface info.
 type Ethernet struct {
 	Match struct {
-		Name   []string `yaml:"name,omitempty"`
```

Comment:
seems like a major change changing to type from a slice to a string, anything that was broken already?

Reply:
i've found this bug just yesterday. I guess this is copy/past issue...
It has to be a string not an array...